packages/reports/app/styles/navi-reports/components/filter-builder.scss

```diff
@@ -47,3 +60,9 @@ $filter-builder-padding: 4px;
     }
   }
 }
+
+.dropdown-parameter-picker {
+  &__dropdown {
+    min-width: 200px !important;
+  }
```

Comment:
Can we color the chip `$links-hover-text-color` on hover and when the menu is open

Reply:
hmm, we should use a darker background too

Reply:
do you have a color in mind?